packages/app/package.json

```diff
@@ -155,6 +155,7 @@
   },
   "devDependencies": {
     "@typescript-eslint/parser": "^2.34.0",
+    "http-proxy-middleware": "^1.0.6",
```

Comment:
we can prob remove [this line](https://github.com/the-difference-engine/ksf/pull/149/files#diff-9fa008865bef344ab2f931a4ddf20163882b86ee764718adacf098dc19991946R5) now?